source/nodejs/adaptivecards/src/controls/utils.ts

```diff
@@ -0,0 +1,7 @@
+export function getScrollX(): number {
```

Comment:
Maybe we can simply put these into the existing `utils.ts` at the root of the `src` directory?